lib/stitches.rb

```diff
@@ -1,4 +1,2 @@
 require 'stitches_norailtie'
 require 'stitches/railtie'
-# Add new stitches ruby files to the stitches_norailtie file instead of here
-require 'apitome'
```

Comment:
This breaks apps generated before this change. They will need to be updated to explicitly require `apitome` in their Gemfile.